mutabazigakuba/Auto_Mart
# #166406948 Built and designed a front-end page for changing price of purchase order

#### What does the PR do?
- It's a front-end design for a user to click and change price of a purchase order
#### Description of Task to be completed?
- Have the link below working
../Auto_Mart/UI/pages/purchases.html
#### How should this be manually tested?
- After cloning the project into your PC, one can visit the link ../Auto_Mart/UI/pages/purchases.html
- then be able to see all the orders and recent purchase
- should be able to view a `change price` link for changing price of a purchase order
#### What are the relevant pivotal tracker stories?
#166406948 [Pivotal Tracker Project](https://www.pivotaltracker.com/n/projects/2352055)
